internal/config/keys.go

```diff
@@ -69,9 +69,12 @@ var DefaultKeyMappings = KeyMappings[keys]{
 		Untrack: []string{"u"},
 	},
 	Git: gitModeKeys[keys]{
-		Mode:  []string{"g"},
-		Push:  []string{"p"},
-		Fetch: []string{"f"},
+		Mode:        []string{"g"},
+		PushFilter:  []string{"p"},
+		PushRevset:  []string{"p", "r"},
+		PushAll:     []string{"p", "a"},
+		FetchFilter: []string{"f"},
+		Fetch:       []string{"f", "f"},
```

Comment:
It's not very clear from the way this keys work but they don't define the sequential key presses, but instead define alternative keys to invoke that item. Only one of them will be processed depending the order of handling in the `Update` function.